test_rustdoc/crate_in_workspace/crate1/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+publish = false
+name = "crate_in_workspace_crate1"
+version = "0.1.0"
```

Comment:
Isnt' this supposed to be `version.workspace = true`?

Reply:
Oh yes sorry for that. I don't know what I'm thinking about yesterday night 🤣